pkg/minikube/constants/constants.go

```diff
@@ -203,16 +135,7 @@ var ImageCacheDir = localpath.MakeMiniPath("cache", "images")
 const (
 	// GvisorFilesPath is the path to the gvisor files saved by go-bindata
 	GvisorFilesPath = "/tmp/gvisor"
```

Comment:
optional: Move all gvisor related constants to `pkg/gvisor`

Reply:
@tstromberg 
When these constants move to `gvisor`, it occurs import cycle incase using these constants  in `assets` package, because `gvisor` has imported `assets` package.
```
= go test ===============================================================
can't load package: import cycle not allowed
package k8s.io/minikube/cmd/gvisor
	imports k8s.io/minikube/pkg/gvisor
	imports k8s.io/minikube/pkg/minikube/assets
	imports k8s.io/minikube/pkg/gvisor
make: *** [test] Error 1
```